remix.init/README-edge.md

```diff
@@ -47,6 +47,25 @@ Open up [http://localhost:3000](http://localhost:3000), and you should be ready
 
 Note: When running the Netlify CLI, file changes will rebuild assets, but you will not see the changes to the page you are on unless you do a browser refresh of the page. Due to how the Netlify CLI builds the Remix App Server, it does not support hot module reloading.
 
+### Excluding routes
```

Comment:
This is a new section I added @stephmarie17. Could the wording be improved?

Reply:
Left a suggestion below!